src/Symfony/Component/Serializer/Tests/Normalizer/UidNormalizerTest.php

```diff
@@ -152,13 +160,23 @@ public function testDenormalize($uuidString, $class)
         $this->assertEquals($class::fromString($uuidString), $this->normalizer->denormalize($uuidString, $class));
     }
 
+    /**
+     * @group legacy
+     */
     public function testDenormalizeOurAbstractUid()
     {
+        $this->expectDeprecation('Since symfony/serializer 6.1: Denormalizing to an abstract class in "Symfony\Component\Serializer\Normalizer\UidNormalizer" is deprecated, it will throw an \Error in 7.0.');
+
         $this->assertEquals(Uuid::fromString($uuidString = '1ea6ecef-eb9a-66fe-b62b-957b45f17e43'), $this->normalizer->denormalize($uuidString, AbstractUid::class));
     }
 
+    /**
+     * @group legacy
+     */
     public function testDenormalizeCustomAbstractUid()
     {
+        $this->expectDeprecation('Since symfony/serializer 6.1: Denormalizing to an abstract class in "Symfony\Component\Serializer\Normalizer\UidNormalizer" is deprecated, it will throw an \Error in 7.0.');
```

Comment:
why is it a legacy test here ? This test does not use `AbstractUid::class`

Reply:
The test uses `TestAbstractCustomUid` that is an abstract class extending `AbstractUid`. All abstract classes cannot be supported.